eng/doc/fips/UserGuide.md

```diff
@@ -240,63 +319,67 @@ func cipher.NewCTR(block Block, iv []byte) (ctr cipher.BlockMode)
 

```

Comment:
Looks like the code block's return part (`ctr cipher.BlockMode`) is wrong:

```
func cipher.NewCTR(block Block, iv []byte) Stream
```

(Can't comment on the line because it didn't actually change in this PR. 😄)

Reply:
Done!